algorithms/src/Kokkos_StdAlgorithms.hpp

```diff
@@ -48,55 +48,91 @@
 /// \file Kokkos_StdAlgorithms.hpp
 /// \brief Kokkos counterparts for Standard C++ Library algorithms
 
-#include <std_algorithms/Kokkos_Constraints.hpp>
-#include <std_algorithms/Kokkos_RandomAccessIterator.hpp>
-#include <std_algorithms/Kokkos_BeginEnd.hpp>
+#include "./std_algorithms/impl/Kokkos_Constraints.hpp"
```

Comment:
Consistency / `#include` style: this is the only place in Kokkos that uses the `./` prefix when dealing with subdirs.
Compare with:
https://github.com/kokkos/kokkos/blob/71c8d24ea3e1eb73b00394ece7a90cd1e6c37e5b/core/src/Kokkos_View.hpp#L57-L59

Consider removing all `./`.

Note: I don't feel strongly about this. Anyone else has an opinion?